rise/static/rise.yaml

```diff
@@ -0,0 +1,216 @@
+  input_type: text
+  default: code
+#
+# a companion to the above - probably not needed
+#
+- name: rise.auto_select_fragment
+  description: >
+    <code>auto_select_fragment</code>: when auto_select is not <code>none</code>, this boolean
+    says if auto selection focuses on the current fragment or considers
+    the whole slide. Probably safe to keep as true.
+  input_type: checkbox
+  default: true
+#
+# headers and footer and background image
+#
+- name: rise.header
+  description: >
+    <code>header</code>: the html text for a header that will show up on all slides.
+    <strong>(effective only if <code>overlay</code> is not defined)</strong>
+  input_type: text
+  default:
+#
+- name: rise.footer
+  description: >
+    <code>footer</code>: the html text for a footer that will show up on all slides.
+    <strong>(effective only if <code>overlay</code> is not defined)</strong>
+  input_type: text
+  default:
+#
+- name: rise.backimage
+  description: >
+    <code>backimage</code>: a (relative) filename for a background image that will show up on all slides.
+    <strong>(effective only if <code>overlay</code> is not defined)</strong>
+  input_type: text
+  default:
+#
+- name: rise.overlay
+  description: >
+    <code>overlay</code>: some html text that will be inserted as-is on all pages. If this is provided,
+    the 3 previous settings (header, footer and backimage) are ignored altogether.
+  input_type: text
+  default:
+#
+# the UI elements for RISE itself
+#
+- name: rise.toolbar_icon
+  description: >
+    <code>toolbar_icon</code>: font-awesome class for the RISE button in toolbar
+  default: fa-bar-chart
+#
+- name: rise.shortcuts.slideshow
+  description: <code>rise.shortcuts.slideshow</code> shortcut to enter and exit slideshow 
+  input_type: hotkey
+  default: alt-r
+- name: rise.shortcuts.toggle-slide
+  description: <code>rise.shortcuts.toggle-slide</code> shortcut to toggle the 'slide' tag on current cell
+  input_type: hotkey
+  default: shift-i
+- name: rise.shortcuts.toggle-subslide
+  description: <code>rise.shortcuts.toggle-subslide</code> shortcut to toggle the 'subslide' tag on current cell
+  input_type: hotkey
+  default: shift-u
+- name: rise.shortcuts.toggle-fragment
+  description: <code>rise.shortcuts.toggle-fragment</code> shortcut to toggle the 'fragment' tag on current cell
+  input_type: hotkey
+  default: shift-f
+- name: rise.shortcuts.toggle-notes
+  description: <code>rise.shortcuts.toggle-notes</code> shortcut to toggle the 'notes' tag on current cell
+  input_type: hotkey
+  default: none
+- name: rise.shortcuts.toggle-skip
+  description: <code>rise.shortcuts.toggle-skip</code> shortcut to toggle the 'skip' tag on current cell
+  input_type: hotkey
+  default: none
+- name: rise.shortcuts.render-all-cells
+  description: <code>rise.shortcuts.render-all-cells</code> shortcut to put all cells in render mode
+  input_type: hotkey
+  default: none
+- name: rise.shortcuts.toggle-skip
```

Comment:
Should not be `toggle.edit-all-cells` below?

Reply:
you are right of course